packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/selectBox.tests.js

```diff
@@ -6481,6 +6481,41 @@ QUnit.module('valueChanged handler should receive correct event', {
     });
 });
 
+QUnit.module('onKeyDown', () => {
+    QUnit.test('should fire event only once (T1238121)', function(assert) {
+        const onKeyDownHandler = sinon.spy();
+        const products = [{
+            ID: 1,
+            Name: 'HD Video Player'
```

Comment:
[minor] imageSrc was an example, do we need ID and Name for tests?

Reply:
hmm let me check it quickly

Reply:
hello alexander based from my investigation its possible to omit ID and Name. so i just removed datasource option and it works as expected thanks.

Reply:
[23ce899](https://github.com/DevExpress/DevExtreme/pull/27660/commits/23ce899856d5ea635e98c13dcfde8280303e3537)